src/__test__/Pages/chartpage.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
- [ ] Your test failed 😢 . Please make sure to fix it as testing is required for this project 👍 
![Capture](https://user-images.githubusercontent.com/6615394/225769003-1f7432d9-c523-4e67-9267-67633dd55d35.PNG)